CHANGELOG.cdf-tk.md

```diff
@@ -17,6 +17,13 @@ Changes are grouped as follows:
 
 ## TBD
 
+### Added
+
+- [Feature Preview] Support for resource type Hosted Extractor Source. This should be in the `hosted_extractors` folder,
+  and the file needs to be suffixed with `.Source.yaml`, for example, `my_eventhub.Source.yaml`.
```

Comment:
using _**.Source.**_ gave me pause, but when we know it sits under hosted_extractors it makes perfect sense 😄